templates/activity.html

```diff
@@ -1,41 +0,0 @@
-<!doctype html>
```

Comment:
please leave the `activity/` area in place.  it will grow.  you can include the one activity widget on the front page to fill the white space, but it also needs to be on `activity/`

Reply:
Okay.